src/main/java/frc/LoggyThings/LoggyTalonFX.java

```diff
@@ -178,28 +239,48 @@ public void set(ControlMode mode, double demand0, DemandType demand1Type, double
 
                 switch (thisLogItem) {
                     case SET_FUNCTION_CONTROL_MODE:
-                        thisEntry.logStringIfChanged(mode.toString(), now);
+                        thisEntry.logStringIfChanged(getControlMode().getValue().toString(), now);
                         break;
                     case SET_FUNCTION_VALUE:
-                        thisEntry.logDoubleIfChanged(demand0, now);
-                        break;
-                    case SET_FUNCTION_DEMAND_TYPE:
-                        thisEntry.logStringIfChanged(demand1Type.toString(), now);
+                        thisEntry.logDoubleIfChanged(get(), now);
```

Comment:
a tostring function wouldnt do anything because their is no tostring override in controlrequest class. so it would just return the ControlRequest@hashcode which isnt really useful, there really is no good way to do this function so maybe I have to create individual cases per control type. idk what your opinion on that is.

Reply:
The child classes do override toString. Not that this matters since we don't want to use this system anyway. And getName() is what we would use to get the name of the control mode.
![image](https://github.com/Robostangs/LoggyThings/assets/29017351/9627b374-826d-42ce-9e17-c51023a6f142)